src/main/java/org/truffleruby/RubyLanguage.java

```diff
@@ -21,15 +21,22 @@
 import com.oracle.truffle.api.source.Source;
 import com.oracle.truffle.api.source.SourceSection;
 import org.jcodings.specific.UTF8Encoding;
+import org.graalvm.options.OptionDescriptor;
 import org.truffleruby.core.kernel.TraceManager;
 import org.truffleruby.core.string.StringOperations;
 import org.truffleruby.language.LazyRubyNode;
 import org.truffleruby.language.LazyRubyRootNode;
 import org.truffleruby.language.RubyGuards;
+import org.truffleruby.options.OptionDescription;
+import org.truffleruby.options.OptionsCatalog;
 import org.truffleruby.platform.Platform;
 import org.truffleruby.stdlib.CoverageManager;
 
+import java.util.ArrayList;
 import java.util.Locale;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
```

Comment:
Quite a few unused imports here and in existing code, it would be to reorganize imports across the codebase.